File: app/src/main/java/com/platdmit/simplecloudmanager/domain/repo/ServerStatisticsRepo.java
package com.platdmit.simplecloudmanager.domain.repo;

import com.platdmit.simplecloudmanager.domain.models.Statistic;

import java.util.List;

import io.reactivex.rxjava3.core.Observable;

public interface ServerStatisticsRepo {
    Observable<List<Statistic>> getServerStatistics(long id);
}

File: app/src/main/java/com/platdmit/simplecloudmanager/domain/repo/ServerStatisticsRepo.kt
package com.platdmit.simplecloudmanager.domain.repo

import com.platdmit.simplecloudmanager.domain.models.Statistic
import io.reactivex.rxjava3.core.Observable

interface ServerStatisticsRepo {
    fun getServerStatistics(id: Long): Observable<List<Statistic>>
}